code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -6,6 +6,12 @@
 		if(shock(user)) //Handles removing charge from the cell, as well. No need to do that here.
 			return
 
+	// Try to use stasis pack
+	if(istype(W, /obj/item/stack/stasis_pack))
```

Comment:
this shouldn't be necessary, there is generic code to make items used on a rig attempt to apply to all of the contained modules, hook into it